Content.Server/GameObjects/Components/Atmos/Piping/PipeComponent.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Content.Server.Atmos;
+using Content.Server.GameObjects.Components.NodeContainer.NodeGroups;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Server.GameObjects.Components.Atmos
+{
+    [RegisterComponent]
+    public class PipeComponent : Component
```

Comment:
Make this a IGasMixtureHolder, please.
AssumeAir simply merges giver mixture into the local mixture.